mikeio/spatial/grid_geometry.py

```diff
@@ -478,7 +480,8 @@ def contains(self, coords):
 
     def find_index(self, x: float = None, y: float = None, coords=None, area=None):
         """Find nearest index (i,j) of point(s)
-           -1 is returned if point is outside grid
+
+        -1 is returned if point is outside grid
```

Comment:
Not exactly part of this PR, but this is not consistent with [list.index](https://docs.python.org/3/tutorial/datastructures.html) which raises ValueError if the value you are looking for is not in the list.

Reply:
Right, I guess the current behavior is better if you have many points (or an area) that you finding indices for...